sources/github/index.js

```diff
@@ -209,3 +209,25 @@ module.exports.formatPreviously = ({ repos, issues, prs, start, end }) => {
 
   return { content, title: `Previously - ${formatDates({ start, end })}` }
 }
+
+module.exports.formatReleases = ({ post, releases, pre = true }) => {
+  if (!releases || releases.length === 0) {
+    return post
+  }
+  let content = `${releases.length} Releases\n`
+  const byProjects = releases.reduce(groupByProj, {})
```

Comment:
what is the reason for passing `groupByProp` this as a prop instead of just writing it inline here?

Reply:
so i know the intent more clearly when i fully refactor underlying data processing using something like pandas for python -- it's too hacky right now